docs/modules/ROOT/pages/how-tos/configuring/build-with-args.adoc

```diff
@@ -0,0 +1,80 @@
+= Passing buildah arguments
+
+{ProductName} buildah task accepts link:https://github.com/konflux-ci/build-definitions/blob/main/task/buildah/0.1/buildah.yaml#L78[`BUILD_ARGS` parameter] that allows passing build arguments to the underlying `buildah` command via `--build-arg` flag. To do this, make the following changes to the PipelineRuns under the `.tekton/` directory:
+
+. Define a `build-args` Pipeline parameter (if not already present) in `.spec.pipelineSpec.params` to accept the passed-in build arguments.
+
++
+[source,yaml]
+--
+spec:
+  # ...
+  pipelineSpec:
+    params:
+    # ...
+    - name: build-args
+      type: array
+      default: []
+--
+
+. Pass the argument to link:https://github.com/konflux-ci/build-definitions/tree/main/task/buildah/[buildah task]. Add `BUILD_ARGS` to `build-container.params` (if not already present):
+
++
+[source,yaml]
+--
+spec:
+  # ...
+  pipelineSpec:
+    tasks:
+    # ...
+    - name: build-container
+      params:
+      # ...
+      - name: BUILD_ARGS
+        value: ["$(params.build-args[*])"]
+--
+
++
+The value can also be in form `tasks.<taskName>.results.<resultName>[*]` if it is generated by another task dynamically.
+
++
+An advanced example of it would be, you can do more creative things with the `BUILD_ARGS` parameter like this snippet of a pipeline that allows passing `build-args` via the pipeline parameter (as described above) but also incorporates `build-args` generated dynamically by other tasks:
```

Comment:
I think the advanced example would work better at the end of the doc. Currently, step 2 is just a too long, and step 3 goes from an advanced example back to something very simple. I'll leave it up to you though

Reply:
Thanks for your suggestion. I'd like to keep it here in order to be compact.